src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -32,6 +32,8 @@ namespace Systems;
 [RuntimeCost(55)]
 public sealed class ProcessSystem : AEntitySetSystem<float>
 {
+    private static readonly Dictionary<BioProcess, float> TempSpeedMultipliers = new();
```

Comment:
I'm pretty sure this is a massive problem as ComputeActiveProcessList has to be thread safe.

Reply:
Would it be right to put all of `ComputeActiveProcessList`'s code in a `lock` statement? Only a single thread would then be able to access the dictionary at a time, though that would worsen performance

Reply:
I wouldn't because this then becomes a bottleneck. If there truly is no way to avoid the temp memory, then the approach where the method caller must provide the working memory should be used (look at for example how the hex adding methods in the hex layout class require multiple working memory parameters to work).

Reply:
This is done now. Each microbe spawn now requires an extra memory allocation, though.